tests/helpers.py

```diff
@@ -1692,8 +1692,8 @@ def verify_pv_mounted_on_node(node_pv_dict):
     """
     existing_pvs = {}
     for node_name, pvs in node_pv_dict.items():
-        cmd = f'oc debug nodes/{node_name} -- df'
-        df_on_node = run_cmd(cmd)
+        cmd = "df"
+        df_on_node = exec_cmd_on_debug_node(node_name, cmd)
```

Comment:
run_cmd should work?

Reply:
It does but this PR is adding `exec_cmd_on_debug_node` for explicitly running commands through the debug node. It's also decorated with a retry to address #2571

Reply:
I missed the retry but then I believe its a test issue @jilju , shouldn't test retry when it knows command can fail? this will cause all commands to retry

Reply:
The issue is not limited to any test case. The error could show up in any occurrence of "oc debug node" command.
Retry only if error is 'Back-off pulling image ..'.

Reply:
@jilju but current implementation cover all the cases of CommandFailed exception which is not anyhow restricted only for this case: Back-off pulling image right?

We have two possibilities:

- update run_cmd function to raise new kind of exception CommandFailedBackOffPullingImage which will inherit from CommandFailed and here do the retry only on that specific exception.
- update the retry function to take additional parameter like message_in_excepion which will look for specific message in the exception and do retry only when `message_in_excepion in str(exception)`

Just my thoughts on this when I saw the reply from Jilju about `Retry only if error is 'Back-off pulling image `

Reply:
@petr-balogh 
Yes, the current implementation will do retry for all cases of CommandFailed exception.
The second possibility you mentioned will be a great enhancement in general.

Reply:
It would be good if we also add a retry for `TLS handshake timeout`  during oc command execution.
https://github.com/red-hat-storage/ocs-ci/issues/1604